org.learningequality.Kolibri.yaml

```diff
@@ -21,6 +22,14 @@ add-extensions:
     subdirectories: true
     no-autodownload: true
 
+add-extensions:
+  org.learningequality.Kolibri.Plugin:
+    version: '1.0'
+    directory: kolibri-plugins
+    merge-dirs: lib/python3.7/site-packages
```

Comment:
One thing to bear in mind here is we'll need to switch to python3.8 at the same time as we switch to the org.gnome.Platform//3.38 runtime, and I expect we'll want to do that fairly soon. Fine for now since we're building the only plugin, but I guess we'd need to either increment the version of this extension point, or keep "/app/lib/python3.7/..." in our PYTHONPATH. Does that sound okay, or should we instead build plugins so they use a generic looking site-packages path?

Gajim, for instance has plugins install to `/app/plugins/${EXTENSION_NAME}/site-packages`: https://github.com/flathub/org.gajim.Gajim/blob/master/org.gajim.Gajim.yaml

Reply:
We've the version number in the `Plugin`, so we can update that when the python version changes. In any case, I've updated the plugin to use a path without the python version.